### PR TITLE
chore(plugin-chart-parallel-coordinates): convert para coords chart to emotion styling

### PR DESCRIPTION
**Pull request by pkdotson**
💔 Breaking Changes

🏆 Enhancements
Convert parallel coordinates chart to emotion styling.

📜 Documentation

🐛 Bug Fix

🏠 Internal

### CODE REVIEW

File: plugins/legacy-plugin-chart-parallel-coordinates/package.json [hunk 1]
```diff
@@ -28,8 +28,10 @@
     "access": "public"
   },
   "dependencies": {
+    "@superset-ui/style": "^0.14.3",
     "d3": "^3.5.17",
-    "prop-types": "^15.6.2"
+    "prop-types": "^15.7.2",
+    "react": "^16.13.1"
```
Comment (ktmud) [inline plugins/legacy-plugin-chart-parallel-coordinates/package.json:34]: Same issue with other plugins.

File: plugins/legacy-plugin-chart-parallel-coordinates/src/index.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ const metadata = new ChartMetadata({
 export default class ParallelCoordinatesChartPlugin extends ChartPlugin {
   constructor() {
     super({
-      loadChart: () => import('./ReactParallelCoordinates.js'),
+      loadChart: () => import('./ReactParallelCoordinates.jsx'),
```
Comment (ktmud) [inline plugins/legacy-plugin-chart-parallel-coordinates/src/index.js:36]: Same here, remove `.jsx`

File: plugins/legacy-plugin-chart-parallel-coordinates/package.json (hunk 1 above)
Comment (ktmud) [reply plugins/legacy-plugin-chart-parallel-coordinates/package.json:33]: react and prop-types should also be peerDependencies.

Comment (kristw) [reply plugins/legacy-plugin-chart-parallel-coordinates/package.json:33]: react is peer but prop-types is dep

Comment (kristw) [reply plugins/legacy-plugin-chart-parallel-coordinates/package.json:33]: Is it really resolved? The code is still the same.

**Pull request merged**